source/ota.c

```diff
@@ -190,6 +190,31 @@ static bool otaClose( OtaFileContext_t * const pFileContext );
 static OtaErr_t setImageStateWithReason( OtaImageState_t stateToSet,
                                          uint32_t reasonToSet );
 
+/* The default OTA callback handler if not provided to OTA_Init(). */
```

Comment:
These files are not used anymore and already deleted.

Reply:
Thanks! Removed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/85/commits/6a8d1b7eee8715f15deecd3fc26ad8f23662a3c7